src/ceph.in

```diff
@@ -109,7 +109,8 @@ def get_cmake_variables(names):
             if line.startswith("{}:".format(name)):
                 vars[name] = line.split("=")[1].strip()
                 break
-        if all(vars.itervalues()):
+        from six import itervalues
+        if all(itervalues(vars)):
```

Comment:
i'd suggest
```suggestion
        if all(vars.values()):
```

to avoid runtime dependency on `python-six` which is now a build-time dependency, and and only a run-time dependency of ceph-mgr. as `vars` is always very small, and we can afford the memory overhead introduced by the temporary list.

Reply:
Done.